eth2/operation_pool/src/lib.rs

```diff
@@ -397,7 +400,9 @@ impl<T: EthSpec> OperationPool<T> {
         self.transfers
             .read()
             .iter()
-            .filter(|transfer| verify_transfer(state, transfer, spec).is_ok())
+            .filter(|transfer| {
+                verify_transfer(state, transfer, VerifySignatures::False, spec).is_ok()
```

Comment:
@michaelsproul we're not verifying signatures here. Please confirm this is desired.

Reply:
Yeah I think it's OK. Previously I think we were checking them here, but it was a double-check because we'd already checked them upon insertion.